tests/foreman/api/test_classparameters.py

```diff
@@ -416,15 +626,25 @@ def test_positive_validate_default_value_with_list(self):
         3. Validate this value with list validator type and rule.
 
         @assert: Error not raised for default value in list.
-
-        @caseautomation: notautomated
-
-        @CaseLevel: System
         """
+        sc_param = self.sc_params_list.pop()
+        sc_param.override = True
+        sc_param.default_value = 'test'
```

Comment:
1) You are saying `pick random value out of it` and then `pass that list`. Value is not a list
2) Here we need to verify that default value that match validator should be good enough for parameter to be updated. It doesn't make sense to introduce unnecessary complexity to the code